CHANGELOG.md

```diff
@@ -1,5 +1,39 @@
 # Mattermost Mobile Apps Changelog
 
+## v1.3 Release 
+
+- Release Date: October 5, 2017
+- Server Versions Supported: Server v4.0+ is required, Self-Signed SSL Certificates are not supported
+
+### Highlights 
+
+#### Tablet Support (Beta) 
+- Added support for landscape view, so the app may be used on tablets
+- Note: Tablet support is in beta, and further improvements are planned for a later date
```

Comment:
yes I think is best to keep it in "Beta" at least to see the response that we get